<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>OptChat</class>
 <widget class="QWidget" name="OptChat" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>OptChatUI</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QCheckBox" name="ck_chatSoftReturn" >
     <property name="text" >
      <string>Enter key sends chat messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_alertOpenChats" >
     <property name="text" >
      <string>Alert chat messages even if chat window is open</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_raiseChatWindow" >
     <property name="text" >
      <string>Raise chat window on receiving new messages</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_smallChats" >
     <property name="text" >
      <string>Open chats in compact mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_tabChats" >
     <property name="text" >
      <string>Open chats in tabbed mode</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="ck_autoResize" >
     <property name="text" >
      <string>Auto-resize text input field</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox18" >
     <property name="title" >
      <string>Default Action</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>-1</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_defActMsg" >
        <property name="text" >
         <string>Message</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_defActChat" >
        <property name="text" >
         <string>Chat</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox17" >
     <property name="title" >
      <string>Delete chat window contents</string>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>10</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QRadioButton" name="rb_delChatsClose" >
        <property name="text" >
         <string>When chat window is closed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_delChatsHour" >
        <property name="text" >
         <string>One hour after chat window was closed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_delChatsDay" >
        <property name="text" >
         <string>One day after chat window was closed</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rb_delChatsNever" >
        <property name="text" >
         <string>Never</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
